airbyte-integrations/connectors/source-avni/source_avni/spec.yaml

```diff
@@ -0,0 +1,23 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Avni Spec
+  type: object
+  required:
+    - username
+    - password
+    - lastModifiedDateTime
+  properties:
+    username:
+      type: string
+      description: Your avni platform Username
+    password:
+      type: string
+      description: Your avni platform password
+      airbyte_secret: true
+    lastModifiedDateTime:
+      type: string
+      default: "2000-06-23T01:30:00.000Z"
+      description: Specify Date and time from which you want to fetch data
+      examples:
+        - "2000-10-31T01:30:00.000Z"
```

Comment:
This should be called `start_date` or `start_replication_date` convention is to use lower case

Reply:
changed to start_date